scripts/build/rollup.config.js

```diff
@@ -145,8 +150,36 @@ export const snippetGTM = {
     file: 'lib/scripts/amplitude-gtm-snippet-min.js',
     format: 'iife',
   },
-  plugins: [
-    createGTMSnippet(),
-    terser(),
-  ],
-};
\ No newline at end of file
+  plugins: [createGTMSnippet(), terser()],
+};
+
+// Input: bookmarklet js template + amplitude js
+// Output: bookmarklet js snippet
+const createBookmarkletSnippet = () => {
```

Comment:
Do we need a related iife?

Reply:
Good question, correct me if I am wrong, I don't think so as this can reuse the [`amplitude-min.js`](https://github.com/amplitude/Amplitude-TypeScript/pull/529/files#diff-eeaf80f697e8ad62106eaf310a1656a71a844d5656021fd0d1cea0c9e4ca2c53R13).

Reply:
I feel the iife is for generating files? Did you try to run yarn build in your local, will that update the existing `/amplitude-min.js` or `amplitude-gtm-min.js`?

Reply:
Yes, `yarn build` will generate the files in `packages/analytics-browser/lib/scripts`, including `amplitude-min.js` and `amplitude-gtm-min.js`. I think [this](https://github.com/amplitude/Amplitude-TypeScript/blob/4dce40d86372a6a4682d688113782939d8d01904/packages/analytics-browser/rollup.config.js#L3) already covers the it.